setup.py

```diff
@@ -35,14 +34,13 @@
         'License :: OSI Approved :: Apache Software License',
         'Natural Language :: English',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.6',
```

Comment:
Btw, is there a specific reason for dropping 3.6?

Reply:
Very few people (that I am aware of) keep on using "old" python versions, most people use either the current stable version (3.8 in this moment) or the previous one (3.7). But apart from that I don't have any other reason

Reply:
I suppose that's fair.
I would recommend keeping `python_requires='>=3.6'` though, at least until we encounter a concrete situation where 3.6 is inadequate (at which point we can fully drop it without a second thought).